lib/404.php

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>404 Not Found</title>
+</head>
+<body>
+	<h1>404 Not Found</h1>
+
+	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
+	<script>window.jQuery || document.write('<script src="public/vendor/jquery.js"><\/script>')</script>
+	<script src="/public/html.js"></script>
+
+	<p>Page not found.</p>
```

Comment:
Inside the `html.php` template, we use an extra constant so that it can be installed at a different URL. We can do the same here, and keep the script tag at the end of the page:

```suggestion
	<p>Page not found.</p>

	<script src="<?php echo EDITH_URI ?>/public/html.js"></script>
```